### DISCUSSION TITLE
Help needed regarding Dialogs

### DISCUSSION THREAD

**Discussion by Pratham95VP** [flarive/Neumorphism.Avalonia#31] [Q&A] [answered]
Can I open a Dialog box on button Click with custom message. Suppose I am performing an operation of uploading or downloading on button click, once the process is finished I want to display the Dialog box.

**Comment by flarive** [comment, marked as answer]
Hello, yes you can...
You can even cascade several dialogs as a dialog sequence, please see samples...
You are free to open dialogs when you want (after upload complete for example), you can even close them by code if needed...

**Reply by Pratham95VP** [reply]
Thank you for the answer. I have gone through the sample / demo for dialogs, but it did not help me in understanding the code and implementation of it in my project as I am new with Avalonia. Can you please provide example here, like I have seen sample code of message box that can be called in c# code like `` await MessageBox.Show(Window, "Message", "Title", MessageBox.MessageBoxButtons.Ok);``

**Comment by Pratham95VP** [comment]
Thanks for the help. I have gone through the demo code once again and found the well written samples.

**Comment by Pratham95VP** [comment]
Thanks for the help. I have gone through the demo code once again and found the well written samples.
